src/main/java/upbrella/be/user/dto/response/SingleHistoryResponse.java

```diff
@@ -0,0 +1,21 @@
+package upbrella.be.user.dto.response;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.Builder;
+import lombok.Getter;
+
+import java.time.LocalDateTime;
+
+@Getter
+@Builder
+public class SingleHistoryResponse {
+
+    private long umbrellaUuid;
+    @JsonFormat(shape= JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd kk:mm:ss")
+    private LocalDateTime rentedAt;
+    private String rentedStore;
+    @JsonFormat(shape= JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd kk:mm:ss")
+    private LocalDateTime returnAt;
+    private boolean isReturned;
+    private boolean isRefunded;
+}
```

Comment:
```suggestion
@JsonFormat(shape= JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd kk:mm:ss")
public class SingleHistoryResponse {

    private long umbrellaUuid;
    private LocalDateTime rentedAt;
    private String rentedStore;
    private LocalDateTime returnAt;
    private boolean isReturned;
    private boolean isRefunded;
}
```

위와 같이 클래스 레벨에 어노테이션을 하나만 사용해도 같은 동작을 할 수 있을 것같은데 어떤 것이 좋아보이나요?